server/api/single-product.js

```diff
@@ -0,0 +1,15 @@
+const router = require('express').Router()
+const {Product} = require('../db/models')
+module.exports = router
+
+// router.use('/singleProduct', require('./singleProduct'))
+router.get('/:id', async (req, res, next) => {
+  try {
+    const product = await Product.findOne({
```

Comment:
If we're getting one would it be more efficient to do findById()?

Reply:
findbyid no longer works but we could do findbypk